### PR TITLE
Added more information about OneNote, Wiki and channels messages in Teams Multigeo

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/3636.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-experience-o365odb-spo-multi-geo.md [hunk 1]
```diff
@@ -23,14 +23,14 @@ appliesto:
...
+Microsoft Teams is group chat software, the hub for teamwork in Microsoft 365 and Office 365. It is powered by the Microsoft 365 Groups service along with SharePoint Online and OneDrive for Business for its files experience. In an OneDrive for Business/SharePoint Online Multi-Geo tenancy, in which the tenant is extended to many geographic locations such as North America, Europe, and Australia, the underlying files experience is Multi-Geo aware, so the Teams experience with file collaboration is also Multi-Geo aware. This is a key leading-edge capability for Teams to surface files hosted across multiple Geos in its native files experience. This also includes OneNote and Wiki files.
```
Comment (JohanFreelancer9) [inline Teams/teams-experience-o365odb-spo-multi-geo.md:26]:
```suggestion
Microsoft Teams is group chat software, the hub for teamwork in Microsoft 365 and Office 365. It is powered by the Microsoft 365 Groups service along with SharePoint Online and OneDrive for Business for its files experience. In a OneDrive for Business/SharePoint Online Multi-Geo tenancy, in which the tenant is extended to many geographic locations such as North America, Europe, and Australia, the underlying files experience is Multi-Geo aware, so the Teams experience with file collaboration is also Multi-Geo aware. This is a key leading-edge capability for Teams to surface files hosted across multiple Geos in its native files experience. This also includes OneNote and Wiki files.
```

Comment (JohanFreelancer9) [reply Teams/teams-experience-o365odb-spo-multi-geo.md:26]: Hi @dariomws - I would suggest leaving the article as-is. Just as a reference, although the first letter of OneDrive is a vowel, it is not pronounced like one, therefore the article should be "a".

File: Teams/teams-experience-o365odb-spo-multi-geo.md [hunk 2]
```diff
@@ -23,14 +23,14 @@ appliesto:
...
 
-Because a Multi-Geo tenancy is a single global tenant, during @ mentions satellite users will be able to see their colleagues from across the globe, no matter where they reside. 
+Because a Multi-Geo tenancy is a single global tenant, during @ mentions satellite users will be able to see their colleagues from across the globe, no matter where they reside.
 
-Note that conversations in chats and meeting IM notes within the Teams experience are not Multi-Geo aware and are all kept only inside the central location of the tenant. Typically, chat conversations aren’t applied to data residency needs.
+Note that conversations in chats, channels messages and meeting IM notes/chats within the Teams experience are not Multi-Geo aware and are all kept only inside the central location of the tenant. Typically, chat conversations aren’t applied to data residency needs. For more information, see [Location of data in Microsoft Teams](location-of-data-in-teams.md).
```
Comment (JohanFreelancer9) [inline Teams/teams-experience-o365odb-spo-multi-geo.md:34]:
```suggestion
Note that conversations in chats, channels messages, and meeting IM notes/chats within the Teams experience are not Multi-Geo aware and are all kept only inside the central location of the tenant. Typically, chat conversations aren’t applied to data residency needs. For more information, see [Location of data in Microsoft Teams](location-of-data-in-teams.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: Teams/teams-experience-o365odb-spo-multi-geo.md (hunk 1 above)
Comment (dariomws) [reply Teams/teams-experience-o365odb-spo-multi-geo.md:26]: Agree

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MicrosoftHeidi PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by MicrosoftHeidi** [comment]
This request needs review by internal resources, and should not be approved at this time. The language of this article is determined by internal resources.

**Pull request merged**